src/styles/Button.module.css

```diff
@@ -1,16 +1,21 @@
 /* Generic component styles */
 
 .btn {
-  background-color: #3333FF;
```

Comment:
Why did you get rid of this?

Reply:
The font color and background color are set dynamically by react. Since these are already listed as default props, there is no need to have them in the CSS.